### PR TITLE
docs: add commit message structure to read me

### PR DESCRIPTION
**Pull request by bexux**
Our commit messages should follow this approach to help trigger the release versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,15 @@ Documentation search is powered by [Algolia's DocSearch](https://community.algol
 
 ## Contributing
 
-Please read through our [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
+Commit messages should follow semantic versioning.
+
+```
+docs: (changes to the documentation)
+feat: (new feature for the user, not a new feature for build script)
+fix: (bug fix for the user, not a fix to a build script)
```
Comment (kalebdf) [inline README.md]:
```suggestion
breaking: (major breaking change - e.g. finally removing float in favor of hovering)
build: (no change - e.g. build script changes that do not affect the final output for the user)
docs: (no change - e.g. changes to the documentation)
feat: (minor change - e.g. new feature for the user, not a new feature for build script)
fix: (patch change - bug fix for the user, not a fix to a build script)
```

File: README.md [hunk 2]
```diff
@@ -82,7 +82,17 @@ Documentation search is powered by [Algolia's DocSearch](https://community.algol
 
 ## Contributing
 
-Please read through our [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
+Commit messages should follow semantic versioning.
+
+```
+breaking: (major breaking change - e.g. finally removing float in favor of hovering)
```
Comment (matttelliott-flosports) [inline README.md]: Also includes changing or removing class names or functionality, and in our case many changes to documentation that break parity with mainline bootstrap.

Comment (bexux) [reply README.md]: Updated

**Review by thatcatcancode — APPROVED** [review]
ACK

**Pull request merged**